test/services/cache/ExportsCacheItemBuilder.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.cache
+
+import java.util.UUID
+
+import models.declaration.ProcedureCodesData
+
+trait ExportsCacheItemBuilder {
```

Comment:
in `ExportsCacheModelBuilder` there's already loads of `withItem(...)` `withItems(...)` helpers that can be extended, I'm not sure we need this

Reply:
Yes we do as that builder is massive and not specific to building items. If we want to test multiplicity we might need easy way to build items, this is that.